### PR TITLE
Update Overview.md

### PR DESCRIPTION
**Pull request by miladpiri**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.

### CODE REVIEW

File: docs/Overview.md [hunk 1]
```diff
@@ -82,6 +82,7 @@ The upgrade itself characterizes by three variables:
 
 Separate facet, whose only function is providing `view` and `pure` methods. It also implements
 [diamond loupe](https://eips.ethereum.org/EIPS/eip-2535#diamond-loupe) which makes managing facets easier.
+This contract, similar to `DiamondCutFaucet` must never be frozen.
```
Comment (vladbochok) [inline docs/Overview.md:85]: We don't have `DiamondCutFaucet` anymore

**Comment by vladbochok** [comment]
@miladpiri We are very time limited, can you please start removing DiamondCutFacet, GovernanceFacet and start writing about AdminFacet, and Governance contract?

File: docs/Overview.md [hunk 2]
```diff
@@ -238,6 +225,16 @@ investigation and mitigation before resuming normal operations.
 It is a temporary solution to prevent any significant impact of the validator hot key leakage, while the network is in
 the Alpha stage.
 
+This contract consists of four main functions `commitBatches`, `proveBatches`, `executeBatches`, and `revertBatches`, that
+can be called only by the validator.
+
+When the validator calls `commitBatches`, the same calldata will be propogated to the zkSync contract (`DiamondProxy` through
+`call` where it invokes the `ExecutorFacet` through `delegatecall`), and also a timestamp is assigned to these blocks to track
+the time these blocks are commited by the validator to enforce a delay between committing and execution of blocks. Then, the
```
Comment (vladbochok) [inline docs/Overview.md:233]: batches

File: docs/Overview.md [hunk 3]
```diff
@@ -56,41 +56,28 @@ diamond constructor and is not saved in the diamond as a facet.
...
+#### AdminFacet
 
 Controls changing the privileged addresses such as governor and validators or one of the system parameters (L2
-bootloader bytecode hash, verifier address, verifier parameters, etc).
+bootloader bytecode hash, verifier address, verifier parameters, etc), and it also manages the freezing/unfreezing and execution of
+upgrades in the diamond proxy.
+
+#### Governance
+
+This contract manages operations (calls with preconditions) for governance tasks. The contract allows for operations to be scheduled,
```
Comment (vladbochok) [inline docs/Overview.md:73]: This is info from top of the file, I would rather want to have description similar to https://www.notion.so/matterlabs/Better-governance-e38da4a92ecb4239bb609ee8e91d0c9c?pvs=4

**Review by StanislavBreadless — APPROVED** [review]

**Review by vladbochok — APPROVED** [review]

**Pull request merged**